go/host/host.go

```diff
@@ -200,13 +200,18 @@ func (h *host) HealthCheck() (*hostcommon.HealthCheck, error) {
 
 // ObscuroConfig returns info on the Obscuro network
 func (h *host) ObscuroConfig() (*common.ObscuroNetworkInfo, error) {
+	l2MessageBusAddress, err := h.EnclaveClient().GetL2MessageBusAddress()
+	if err != nil {
+		return nil, responses.ToInternalError(fmt.Errorf("unable to get L2 message bus address - %w", err))
+	}
```

Comment:
Should we have a generic enclave config  method instead of the specific GetL2MessageBusAddress ? Perhaps I'm future-proofing..